doc/source/user_guide/ug_testing.rst

```diff
@@ -229,7 +229,9 @@ boundary around the entire domain.  It includes the following namelist modificat
 - ``dyrect``: ``16.e5`` cm
 - ``ktherm``: ``-1`` (disables thermodynamics)
 - ``coriolis``: ``constant`` (``f=1.46e-4`` s\ :math:`^{-1}`)
-- ``ice_data_type`` : ``box2001`` (special ice concentration initialization)
+- ``ice_data_type`` : ``box2001`` (special initial ice mask)
```

Comment:
If I remember correctly, the 'box2001' land mask in the code does not have the 3 islands that were used in the original paper, which I think means that the land mask now is simply closed around all of the sides.  Is that correct?